sda-commons-server-testing/src/test/java/org/sdase/commons/server/testing/SystemPropertyClassExtensionTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sdase.commons.server.testing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class SystemPropertyClassExtensionTest {
+
+  @RegisterExtension
+  @Order(0)
+  static final TestWrapper WRAPPER = new TestWrapper();
```

Comment:
What about a better name like "PreAndPostConditions" for class (and the variable)? :)